examples/simple_project_with_regional_network/versions.tf

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright 2018 Google LLC
```

Comment:
For newly added copyrights, we probably want it to be 2019, not 2018.

Reply:
See https://github.com/terraform-google-modules/terraform-google-network/pull/54